04/index.html

```diff
@@ -1,30 +1,37 @@
 <!DOCTYPE html>
 <html lang="pl">
+
 <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>devmentor.pl - HTML & CSS: RWD - #04</title>
+    <link rel="stylesheet" href="styles/style.css">
 </head>
+
 <body>
-    
-    <nav>
-        <ul>
-            <li>
-                <a href="#">start</a>
-            </li>
-            <li>
-                <a href="#">oferta</a>
-                <ul>
-                    <li><a href="">strony internetowe</a></li>
-                    <li><a href="">pozycjonowanie stron www</a></li>
-                </ul>
-            </li>
-            <li>
-                <a href="#">kontakt</a>
-            </li>
-        </ul>
-    </nav>
+    <header>
+        <nav class="nav">
+            <label for="menu-switcher"></label>
+            <input type="checkbox" id="menu-switcher">
+            <ul class="nav__links">
+                <li class="nav__link">
+                    <a href="#">start</a>
+                </li>
+                <li class="nav__link nav__link--dropdown">
+                    <a href="#">oferta</a>
+                    <ul class="nav__dropdown">
+                        <li class="dropdown__link"><a href="">strony internetowe</a></li>
```

Comment:
Dla elementu BEM o nazwie "dropdown__link" brakuje w strukturze bloku o nazwie "dropdown", można to tak zrobić:
```
<ul class="nav__dropdown dropdown">
    <li class="dropdown__link"><a href="">strony internetowe</a></li>
```